### ISSUE TITLE
The pagenum variable is being appended to the letter links

### ISSUE THREAD

**Issue by rafaehlers** [gravityview/GravityView-AZ-Filters#17]
>However when viewing an additional page after p 1 for a given letter and then clicking from there to a new letter, the user lands on the second page of that new letter. So for example, go to letter A and then to the second page. From there click letter H. Letter H only has one page of submissions, yet the user lands on this URL: https://example.com/applications/?pagenum=2&letter=h. Note pagenum=2 there. Can you suggest a way for the search to be reset so the user lands on the first page of the new letter? Thanks!

It looks like this happens because we're using the add_query_arg to URL:

On `gravityview-a-z-entry-filter-widget.php`:
```
// If entries exist then change the link for the letter.
			if( $char === $args['number_character'] ) {
				$link = add_query_arg( array( $this->letter_parameter => '0-9' ) );
				$title = $args['link_title_number'];
			} else {
				$link = add_query_arg( array( $this->letter_parameter => $char ) );
				$title = sprintf( $args['link_title_letter'], $char );
			}
```

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/300740581/7814/?folderId=408730

**Comment by soulseekah** [comment]
Fixed in 2.0. Push customers towards the beta :)
